pinot-udf/build.gradle.kts

```diff
@@ -11,6 +11,6 @@ configure<JavaPluginExtension> {
 }
 
 dependencies {
-  implementation("org.hypertrace.core.attribute.service:attribute-projection-functions:0.3.4-SNAPSHOT")
+  implementation("org.hypertrace.core.attribute.service:attribute-projection-functions:0.4.1")
```

Comment:
any chance of some weird cyclin dependency?

Reply:
Nope - we don't publish anything for pinot and attribute projection functions intentionally has minimal dependencies. If the projection adds dependencies, they're all added to pinot - so those need to kept very minimal and if using any common libs we would need to shade them.